purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/properties/Shadow.kt

```diff
@@ -5,7 +5,7 @@ import kotlinx.serialization.Serializable
 
 @InternalRevenueCatAPI
 @Serializable
-internal data class Shadow(
+data class Shadow(
```

Comment:
Should this also use `Poko` instead of being a data class?

Reply:
Good catch! https://github.com/RevenueCat/purchases-android/pull/1952/commits/ae8140c9634e7bca722714bf24a16f20204c946f